main.css

```diff
@@ -0,0 +1,83 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding-left: 2px;
+}
+
+body {
+  position: absolute;
+  width: 375px;
+  height: 590px;
+  background: #1c1a19;
+}
```

Comment:
- [ ] These properties applied to the body make us not achieve the desired design from the Figma template. There is no need to constrain the height and width of the ```<body>```. We expect the page to stretch the whole width in mobile design. Take a look at the following screenshots and see how our page isn't well aligned with the viewport.

### Implementation
![implementation](https://user-images.githubusercontent.com/45002654/124970041-1cbafa80-e030-11eb-8a14-764b4c9094e4.PNG)


### Figma Design
![design](https://user-images.githubusercontent.com/45002654/124970102-2ba1ad00-e030-11eb-99f3-77994640ffbf.PNG)